src/lib/mongoBackend/MongoCommonRegister.h

```diff
@@ -49,4 +50,76 @@ extern HttpStatusCode processRegisterContext
   const std::string&        fiwareCorrelator
 );
 
+
+
+/* ****************************************************************************
+*
+* mongoRegistrationIdExtract -
+*/
+extern void mongoRegistrationIdExtract(ngsiv2::Registration* regP, const mongo::BSONObj& r);
```

Comment:
mongo*Extract() function should return to set*() with the . Changing names and generalization (i.e. dbLabel) would be done in the future. An issue could be opened regarding that.

Reply:
To do that ... I need to make the functions static, which means, in order to reuse the functions, I have to put two external functions in the same file. Not my cup of tea, but to pick the least bad solution, that's what I will do. All "set*" functions go back to being static and both GET methods in the same file

Reply:
Done